src/main/java/org/jabref/logic/importer/fileformat/EndnoteXmlImporter.java

```diff
@@ -107,9 +107,10 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                 // Check whether we have an article set, an article, a book article or a book article set
                 Xml root = (Xml) unmarshalledObject;
                 List<BibEntry> bibEntries = root.getRecords()
-                                                .getRecord().stream()
-                                                .map(this::parseRecord)
-                                                .collect(Collectors.toList());
+                        .getRecord()
```

Comment:
@Siedlerchr Thanks for the fix! However, it appears there is still a problem with your eclipse style configuration (alignment at dot).